### ISSUE TITLE
aiida-quantumespresso no longer works with QE 6.3 on Daint

### ISSUE THREAD

**Issue by ltalirz** [aiidalab/aiidalab-cscs#7]
CSCS recently dropped the module for Quantum ESPRESSO 6.1
```
tal@daint103:~> module avail Qu

- /apps/daint/UES/jenkins/6.0.UP07/mc/easybuild/modules/all --
QuantumESPRESSO/6.3-CrayIntel-18.08(default)
```

The QE 6.3 they provide was (I guess) compiled without the `-D__OLDXML` option, and `aiida-quantumespresso` fails:

```
|   The xml output file 'data-file.xml' was not found but is required
```

@giovannipizzi @sphuber
Should we ask them to compile a 6.3 with `-D__OLDXML`?

Alternatively, the 6.1 binary is still there (but no module):
```
/apps/daint/UES/jenkins/6.0.UP04/gpu/easybuild/software/QuantumESPRESSO/6.1.0-CrayIntel-17.08/
```

**Comment by ltalirz** [comment]
In the meanwhile, who will be working on the adaptation of aiida-quantumespresso to the new xml format?
Is this something where we could ask Casper for help?

**Comment by sphuber** [comment]
I have written the first implementation of adapting the parser to supporting the new XML format together with the schema together with @giovannipizzi . The bulk of the work has been done, but now we need to set up tests to ensure that we produce the same output nodes as with the old parsers, as well as making sure that we add any additional interesting information that is written in the new XML. I have brought @borellim up to speed who should be finalizing this work.

**Comment by ltalirz** [comment]
I've written to CSCS in the meanwhile

**Comment by ltalirz** [comment]
fixed in PR #8
